js/web/lib/wasm/jsep/webgpu/types.ts

```diff
@@ -12,6 +12,12 @@ export enum GpuDataType {
 }
 export type GpuDataId = number;
 
+export enum QueryType {
+  none,
+  insidePasses,
+  atPasses,
+}
```

Comment:
Use `const enum` or string literals for enum. Typescript generates a few JS code only for compatibility of old typescript enum standard and is not encouraged to use.

Reply:
Thanks for the suggestion. Done.